Logs to metrics (#4990)
DataDog/documentation: content/en/logs/processing/logs_to_metrics.md

With log-based metrics, generate custom metrics from the count of logs that match a filter query and apply log attributes and tag keys as [metric tags][3] following the `<KEY>:<VALUE>` format.

## Generate a log-based metric

**Creating and editing log-based metrics is limited to admin users.**

{{< img src="logs/processing/logs_to_metrics/generate_logs_to_metric.png" alt="Generate Logs to metric" responsive="true" style="width:80%;">}}

To generate a new log-based metric, go to the [Configuration page][4] of your Datadog account and select the *[Generate Metrics][5]* tab, then the **New Metric+** button.

### Add a new log-based metric

{{< img src="logs/processing/logs_to_metrics/create_custom_metrics.png" alt="Create a Logs to metric" responsive="true" style="width:80%;">}}

1. **Input a query to filter the log stream**: The query syntax is the same as for the [Log Explorer Search][6]. Only logs ingested with a timestamp within the past 20 minutes are considered for aggregation.

> 2. **Add dimensions to `group by`**: Add attributes or tag key to apply to the generated log-based metric to transform them into [Tags][3] following the `<KEY>:<VALUE>` format. Log-based metrics are considered as [custom metrics][7]. So, avoid grouping by unbounded or extremely high cardinality attributes like timestamps, user IDs, request IDs, and session IDs to avoid impacting your billing.

Reviewer: ```suggestion
2. **Add dimensions to `group by`**: Add attributes or tag key to apply to the generated log-based metric to transform them into [Tags][3] following the `<KEY>:<VALUE>` format. Log-based metrics are considered as [custom metrics][7]. Avoid grouping by unbounded or extremely high cardinality attributes like timestamps, user IDs, request IDs, or session IDs to avert impacting your billing.
```
